docs/guides/verify-lido-v2-upgrade-manual.md

```diff
@@ -82,8 +82,8 @@ NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100  # 1% network penetration for
 PREDICTION_DURATION_IN_SLOTS=50400  # 7 days
 
 # Max period of delay for requests finalization in case of bunker due to negative rebase
-# twice min governance response time - 3 days voting duration
-FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+# 2 x the min governance reaction time (5 days voting duration)
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=2250  # 10 days
```

Comment:
there's no need in updating this doc as it had to be relevant at the point of V2 updrage